frontend/src/features/workspace/WorkspacePage.tsx

```diff
@@ -122,63 +138,19 @@ export const WorkspacePage = (): JSX.Element => {
     [isUserLoading, hasSeenAnnouncement],
   )
 
+  // TODO (hans): Get totalFormCount and isLoading from GET workspaces API when it is implemented
+  const totalFormCount = Math.round(Math.random())
```

Comment:
/nit even for a mock, get a bigger range of number, maybe? the current code only gives 0 or 1 😅 (or is the 0/1 on purpose?)

```suggestion
  const totalFormCount = Math.round(Math.random() * 1000)
```

Reply:
Updated this to use `msw` to mock, this is not required anymore!